tendrl-node-agent.spec

```diff
@@ -1,5 +1,5 @@
 Name: tendrl-node-agent
-Version: 1.2
+Version: 0.0
```

Comment:
oops, i changed the version for testing the rpm, forget to revert back. I will do revert this change.